src/ServiceControl/CompositeViews/Messages/GetMessagesController.cs

```diff
@@ -1,15 +1,24 @@
 namespace ServiceControl.CompositeViews.Messages
 {
+    using ServiceControl.Operations.BodyStorage.Api;
     using System.Net.Http;
     using System.Threading.Tasks;
     using System.Web.Http;
 
     public class GetMessagesController : ApiController
```

Comment:
Should we mention in a comment that messages routes need to be on this controller to not mess up catch all?

Reply:
Added comment